process/coordinator/process_test.go

```diff
@@ -199,20 +199,20 @@ func initAccountsMock() *mock.AccountsStub {
 
 func createMockTransactionCoordinatorArguments() ArgTransactionCoordinator {
 	argsTransactionCoordinator := ArgTransactionCoordinator{
-		Hasher:               &mock.HasherMock{},
-		Marshalizer:          &mock.MarshalizerMock{},
-		ShardCoordinator:     mock.NewMultiShardsCoordinatorMock(5),
-		Accounts:             &mock.AccountsStub{},
-		MiniBlockPool:        testscommon.NewPoolsHolderMock().MiniBlocks(),
-		RequestHandler:       &mock.RequestHandlerStub{},
-		PreProcessors:        &mock.PreProcessorContainerMock{},
-		InterProcessors:      &mock.InterimProcessorContainerMock{},
-		GasHandler:           &mock.GasHandlerMock{},
-		FeeHandler:           &mock.FeeAccumulatorStub{},
-		BlockSizeComputation: &mock.BlockSizeComputationStub{},
-		BalanceComputation:   &mock.BalanceComputationStub{},
-	EconomicsFee: 		&mock.FeeHandlerStub{},
-		TxTypeHandler: &mock.TxTypeHandlerMock{},
+		Hasher:                            &mock.HasherMock{},
+		Marshalizer:                       &mock.MarshalizerMock{},
+		ShardCoordinator:                  mock.NewMultiShardsCoordinatorMock(5),
+		Accounts:                          &mock.AccountsStub{},
+		MiniBlockPool:                     testscommon.NewPoolsHolderMock().MiniBlocks(),
+		RequestHandler:                    &mock.RequestHandlerStub{},
+		PreProcessors:                     &mock.PreProcessorContainerMock{},
+		InterProcessors:                   &mock.InterimProcessorContainerMock{},
+		GasHandler:                        &mock.GasHandlerMock{},
+		FeeHandler:                        &mock.FeeAccumulatorStub{},
+		BlockSizeComputation:              &mock.BlockSizeComputationStub{},
+		BalanceComputation:                &mock.BalanceComputationStub{},
+		EconomicsFee:                      &mock.FeeHandlerStub{},
```

Comment:
too many spaces

Reply:
It is ok, if you look below at line 216 you find this line:
BlockGasAndFeesReCheckEnableEpoch: 0,